crates/vm/backends/mod.rs

```diff
@@ -37,6 +38,7 @@ impl TryFrom<String> for EvmEngine {
     }
 }
 
+#[allow(clippy::large_enum_variant)]
 pub enum Evm {
     REVM {
         state: EvmState,
```

Comment:
Clippy suggested using:

```suggestion
pub enum Evm {
    REVM {
        state: EvmState,
    },
    LEVM {
        store_wrapper: Box<StoreWrapper>,
        block_cache: Box<CacheDB>,
    },
}
```

Should I go for the suggestion?